WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderCommentAdapter.java

```diff
@@ -279,16 +280,32 @@ public void onClick(View view) {
         if (mIsLoggedOutReader) {
             commentHolder.txtReply.setVisibility(View.GONE);
             commentHolder.imgReply.setVisibility(View.GONE);
-        } else if (mReplyListener != null) {
-            // tapping reply icon tells activity to show reply box
-            View.OnClickListener replyClickListener = new View.OnClickListener() {
-                @Override
-                public void onClick(View v) {
-                    mReplyListener.onRequestReply(comment.commentId);
-                }
-            };
-            commentHolder.txtReply.setOnClickListener(replyClickListener);
-            commentHolder.imgReply.setOnClickListener(replyClickListener);
+        } else {
+            if (mReplyListener != null) {
+                // tapping reply icon tells activity to show reply box
+                View.OnClickListener replyClickListener = new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        mReplyListener.onRequestReply(comment.commentId);
+                    }
+                };
+                commentHolder.txtReply.setOnClickListener(replyClickListener);
+                commentHolder.imgReply.setOnClickListener(replyClickListener);
+            }
+
+            if (mDoLikeCommentId != 0 && mDoLikeCommentId == comment.commentId) {
```

Comment:
I'm really not wild about leaving this the way it is, but I'm okay with you filing a separate issue for it and tackling it in another PR.